gpcontrib/arenadata_toolkit/arenadata_toolkit--1.0--1.1.sql

```diff
@@ -1,3 +1,45 @@
 /* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.0--1.1.sql */
 
 ALTER FUNCTION arenadata_toolkit.adb_get_relfilenodes(tablespace_oid OID) ROWS 30000000;
+
+CREATE OR REPLACE FUNCTION arenadata_toolkit.adb_rel_storage_size_on_seg(reloid OID, forkName TEXT default 'main')
```

Comment:
I think you should call this function adb_relation_storage_size_segment_location.

Reply:
I minimized length of function's name to minimize difference in the tests output.